lib/i18n/tests/lookup.rb

```diff
@@ -21,6 +21,10 @@ def setup
         assert_equal(%w(a b c), I18n.t(:array))
       end
 
+      test "lookup: it returns the interpolate_array" do
+        assert_equal ['foo', '%{bar}'], I18n.t(:interpolate_array)
+      end
```

Comment:
There's already a test for an `Array` and since an "interpolation array" is no different, I don't see any value in this test.